### ISSUE TITLE
Installation is not relocatable

### ISSUE THREAD

**Issue by mattgodbolt** [carbon-language/carbon-lang#1413]
Having build and installed using something like [this](https://github.com/compiler-explorer/carbon-builder/blob/dcc95ccb4eae43e74e680dd060d406ac8d13278f/build/build.sh#L47-L50) the resulting symlinks appear to be absolute.

When tarr-ing the results of the above and then unpacking on another machine:

```
~/d/c/infra (main|…) $ ls -l /opt/compiler-explorer/carbon-trunk-20220719/bin/carbon-explorer
lrwxrwxrwx 1 matthew matthew 37 Jul 19 09:03 /opt/compiler-explorer/carbon-trunk-20220719/bin/carbon-explorer -> /home/build/staging/lib/carbon/carbon
```

the symlinks are pointing at the directory the build was done in. (the `${STAGING_DIR}` of the linked script)

**Comment by jonmeow** [comment]
Should be fixed by #1427

**Comment by mattgodbolt** [comment]
Thanks @jonmeow !
